tasks.rb

```diff
@@ -1368,11 +1369,20 @@ def set_hpxml_walls(hpxml_file, hpxml)
       wall.solar_absorptance = nil
       wall.color = HPXML::ColorMedium
     end
+  elsif ['base-enclosure-common-walls.xml'].include? hpxml_file
+    hpxml.walls.add(id: 'CommonWallCrawlspace',
+                    exterior_adjacent_to: HPXML::LocationCrawlspaceVented,
+                    interior_adjacent_to: HPXML::LocationCrawlspaceVented,
+                    wall_type: HPXML::WallTypeWoodStud,
+                    area: 113,
+                    solar_absorptance: 0.7,
+                    emittance: 0.92,
+                    insulation_assembly_r_value: 4)
   end
   hpxml.walls.each do |wall|
     next unless wall.is_interior
 
-    fail "Interior wall '#{wall.id}' in #{hpxml_file} should not have siding." unless wall.siding.nil?
+    fail "Common wall '#{wall.id}' in #{hpxml_file} should not have siding." unless wall.siding.nil?
```

Comment:
I don't agree with this change. It applies to all walls where `is_interior` is true, not just walls where InteriorAdjacentTo=ExteriorAdjacentTo.